openfe/protocols/openmm_rfe/equil_rfe_methods.py

```diff
@@ -267,24 +276,25 @@ def get_estimate(self) -> unit.Quantity:
         """
         # TODO: Check this holds up completely for SAMS.
         dGs = [pus[0].outputs['unit_estimate'] for pus in self.data.values()]
+        return self.compute_mean_estimate(dGs)
+
+    @staticmethod
+    def compute_uncertainty(dGs:list):
```

Comment:
This should have the same signature as above.

Also we should possible rename this to "compute_stdev" - just because "uncertainty" is highly undefined.